modules/openapi-generator/src/main/resources/JavaJaxRS/cxf/api.mustache

```diff
@@ -33,7 +33,7 @@ import javax.validation.Valid;
  */
 {{/appName}}
 @Path("{{^useAnnotatedBasePath}}/{{/useAnnotatedBasePath}}{{#useAnnotatedBasePath}}{{contextPath}}{{/useAnnotatedBasePath}}")
-@Api(value = "/", description = "{{description}}")
+@Api("/")
```

Comment:
@demonfiddler shall we keep the description?

Reply:
> some ugly annotations

Should it be using `{{{description}}}` (unescaped text) instead to avoid the ugly annotations?

Reply:
-1 for removing the description.

Users who generate the open-api spec based on the annotation value (at runtime for example) will be impacted by the change.

Reply:
@jmini - on reflection, agreed. My latest push reinstated the description.